Jenkinsfile

```diff
@@ -8,8 +8,10 @@ def apps = [
   [constantPrefix: "CALENDARS", app: "calendars", name: "Calendars"],
   [constantPrefix: "COLLECTIONS", app: "collections", name: "Collections"],
   [constantPrefix: "COLLECTIONS_PUBLISHER", app: "collections-publisher", name: "Collections Publisher"],
+  [constantPrefix: "CONTACTS", app: "contacts-admin", name: "Contacts"],
```

Comment:
Probably should go Contacts Admin for each of these, as that seems to be the canonical name